xbmc/cores/AudioEngine/Utils/AEStreamInfo.cpp

```diff
@@ -446,6 +446,25 @@ bool CAEStreamParser::TrySyncAC3(uint8_t *data, unsigned int size, bool resyncin
     m_fsize = framesize << 1;
     m_info.m_repeat = MAX_EAC3_BLOCKS / blocks;
 
+    /* EAC3 can have a dependent stream too */
+    if (!wantEAC3dependent)
+    {
+      unsigned int fsizeMain = m_fsize;
```

Comment:
f as in frame? Maybe `frameSizeMain`?

Reply:
Rest of file uses f instead of frame, too.

Reply:
This code is pretty much a copy of the code above just in the EAC3 path, rather than AC3 path.

Reply:
So is that reason enough not to update new code to the current guidelines? Just because it's copy paste?

Reply:
just my 2c: I'd say before you enforce new coding styles just bulk-apply that style to your existing code so it meets that criteria, then copy&paste isn't a problem.

If you don't do it a single file will contain a mixture of all coding styles you had over time and will be really unpleasant to read.

Reply:
> just my 2c: I'd say before you enforce new coding styles just bulk-apply that style to your existing code so it meets that criteria, then copy&paste isn't a problem.
> 
> 
> 
> If you don't do it a single file will contain a mixture of all coding styles you had over time and will be really unpleasant to read.

We decided a few years ago that this wasn't the route that we wanted to take. I don't remember all the details at the moment.

Reply:
> So is that reason enough not to update new code to the current guidelines? Just because it's copy paste?

No. It makes zero sense though to make the semantic, with half the file using f less intuitive by enforcing regulatoric enhancements. This drives the whole idea behind that guidelines nuts and decreases the semantic understanding. Let the change in and make a refactor afterwards for entire file to rename f to frame. This allows backports and really helps towards the well meant guidelines.

Reply:
> > So is that reason enough not to update new code to the current guidelines? Just because it's copy paste?
> 
> 
> 
> No. It makes zero sense though to make the semantic, with half the file using f less intuitive by enforcing regulatoric enhancements. This drives the whole idea behind that guidelines nuts and decreases the semantic understanding. Let the change in and make a refactor afterwards for entire file to rename f to frame. This allows backports and really helps towards the well meant guidelines.

Sure, leave the variable name. The rest should be updated to follow the guidelines and clang-format though.

Promise of refactoring later never works.

Reply:
Yes - I agree. Thanks much.